lib/shopify-cli/messages/messages.rb

```diff
@@ -3,6 +3,16 @@
 module ShopifyCli
   module Messages
     MESSAGES = {
+      project: {
```

Comment:
Should we consider namespacing these under `apps` rather than `project`? Not sure if these apply to scripts and extensions.

Reply:
Yes good point, I'll update.

Reply:
and if you do rename it to `app`, you don't have to move it to make this list alphabetical!